tests/binary.rs

```diff
@@ -40,16 +39,19 @@ mod binary {
     Command::cargo_bin("rav1e").unwrap()
   }
 
+  fn get_common_cmd(outfile: &str) -> Command {
+    let mut cmd = get_rav1e_command();
+    cmd.args(&["--bitrate", "1000"]).args(&["-o", outfile]);
```

Comment:
Why not factoring in also 
``` rust
      .arg("-")
      .write_stdin(get_y4m_input())
      .assert()
```

Reply:
Some uses of the fn need to add another arg before the `-` (e.g. [here](https://github.com/xiph/rav1e/pull/2344/files#diff-b1aba0aa09c8f43383202cbf5085a064R78), so I think I can't add it already in the function.

Also I understood, that after the call to `.assert()` the `Command` get's executed, so it should only be called after all the args are added.

Reply:
You are right